### PR TITLE
Bumping version to 0.2.6

### PR DESCRIPTION
**Pull request by john-science**
## Description

This is simply a version bump to `0.2.6`; no code changes are made here.

This version bump is happening because a LOT of work has been done in ARMI lately. We have added several new features and tested and polished them.

I have tried to clean up our massive list of release notes for this version, though it is a big task. In the future, I hope to do a better job making sure the release notes are high-level readable for people on the outskirts of the ARMI ecosystem. The biggest features at the top of the list, with shorter explanations, and more consistent references to PRs.

---

## Checklist

- [X] This PR has only one purpose or idea.
- [X] Tests have been added/updated to verify that the new/changed code works.

- [X] The code style follows [good practices](https://terrapower.github.io/armi/developer/standards_and_practices.html).
- [X] The commit message(s) follow [good practices](https://terrapower.github.io/armi/developer/tooling.html).

- [X] The [release notes](https://terrapower.github.io/armi/release/index.html) (location `doc/release/0.X.rst`) are up-to-date with any bug fixes or new features.
- [X] The documentation is still up-to-date in the `doc` folder.
- [X] The dependencies are still up-to-date in `setup.py`.

### CODE REVIEW

**Review by keckler — APPROVED** [review]

File: doc/release/0.2.rst [hunk 1]
```diff
@@ -2,57 +2,65 @@
...
-#. Axially expand from cold to hot before deepcopy of assemblies into reactor improving speed and preserving same functionality. (`PR#1047 <https://github.com/terrapower/armi/pull/1047>`_)
-#. Add a how-to on restart calculations in the docs
-#. Overhaul of the tight coupling routine in ARMI. Removal of ``looseCoupling`` setting and introduction of new tight coupling user interface. (`https://github.com/terrapower/armi/pull/1033`_)
-#. General improvements to efficiency in many places to speed up uniform mesh conversion. (`PR#1042 <https://github.com/terrapower/armi/pull/1042>`_)
+#. Added 4614 nuclides to decouple the loading of RIPL-3 data from the standard framework run. (`PR#998 <https://github.com/terrapower/armi/pull/998>`_)
+#. Overhaul of the tight coupling routine in ARMI, and removal of ``looseCoupling`` setting. (`https://github.com/terrapower/armi/pull/1033`_)
+#. Added ``savePhysicsFiles`` setting to copy physics kernel I/O to directories organized by cycle and time step (e.g., c2n1). (`PR#958 <https://github.com/terrapower/armi/pull/952>`_)
```
Comment (albeanth) [inline doc/release/0.2.rst:32]: The PR hyperlink and tag(?) don't match here. 

@mgjarrett was this one of your PRs?

**Review by albeanth — CHANGES_REQUESTED** [review]

File: doc/release/0.2.rst [hunk 2]
```diff
@@ -2,57 +2,65 @@
...
-#. Fixed a bug in which the TD_frac material modification on UraniumOxide and MOX materials was not being applied correctly in density calculations
-#. Added hotChannelFactors assembly parameter to the DB so that snapshot runs know about the parameter. (`PR#1157 <https://github.com/terrapower/armi/pull/1157>`_)
+#. Fixed ``referenceBlockAxialMesh`` and ``axialMesh`` during process loading. (`PR#980 <https://github.com/terrapower/armi/pull/980>`_)
+#. Fixed deadelines in MPI cases due to barriers in temp directory changers.
+#. Fixed the material namespace order for ``test_axialExpansionChanger.py`` persisting after tests. (`PR#1046 https://github.com/terrapower/armi/pull/1046`_)
+#. Fixed the gaseous fission products not being removed from the core directly, but instead the fission yields within the lumped fission products were being adjusted. (`PR#1022 <https://github.com/terrapower/armi/pull/1022>`_)
+#. Fixed non-fuel depletable components not being initialized with all nuclides with the ``explicitFissionProducts`` model (`PR#1067 https://github.com/terrapower/armi/pull/1067`_)
```
Comment (opotowsky) [inline doc/release/0.2.rst:53]: same as below

File: doc/release/0.2.rst [hunk 3]
```diff
@@ -2,57 +2,65 @@
...
-#. Bug fix for Concrete density curve. (`PR#1131 https://github.com/terrapower/armi/pull/1131`_)
-#. Bug fix for Copper density curve. (`PR#1150 https://github.com/terrapower/armi/pull/1150`_)
-#. Bug fix for Component.density. (`PR#1149 https://github.com/terrapower/armi/pull/1149`_)
+#. Refine logic for ``Block.getNumPins()`` to only count components that are actually pins. (`PR#1098 <https://github.com/terrapower/armi/pull/1098>`_)
+#. Improve handling of peak/max parameters by the ``UniformMeshConverter`` parameter mapper. (`PR#1108 <https://github.com/terrapower/armi/pull/1108>`_)
 #. Calculate block kgHM and kgFis on core loading and after shuffling. (`PR#1136 https://github.com/terrapower/armi/pull/1136`_)
-#. Update point DPAs along with block aveage DPA values. (`PR#1156 https://github.com/terrapower/armi/pull/1156`_)
-#. Calculate block PuFrac on core loading and after shuffling. (`PR#1165 https://github.com/terrapower/armi/pull/1165`_)
-#. Fix the formula to calculate b.p.puFrac. (`PR#1168 https://github.com/terrapower/armi/pull/1168`_)
+#. Calculate block ``PuFrac`` on core loading and after shuffling. (`PR#1165 https://github.com/terrapower/armi/pull/1165`_)
```
Comment (opotowsky) [inline doc/release/0.2.rst:45]: ditto below

File: doc/release/0.2.rst [hunk 4]
```diff
@@ -2,57 +2,65 @@
...
-#. An explicit fission product modeling option was added in `PR#1022 <https://github.com/terrapower/armi/pull/1022>`_.
-#. Cleanup of unnecessary and confusing functionality in axialExpansionChanger.py. (`PR#1032 <https://github.com/terrapower/armi/pull/1032>`_)
-#. Axially expand from cold to hot before deepcopy of assemblies into reactor improving speed and preserving same functionality. (`PR#1047 <https://github.com/terrapower/armi/pull/1047>`_)
-#. Add a how-to on restart calculations in the docs
-#. Overhaul of the tight coupling routine in ARMI. Removal of ``looseCoupling`` setting and introduction of new tight coupling user interface. (`https://github.com/terrapower/armi/pull/1033`_)
-#. General improvements to efficiency in many places to speed up uniform mesh conversion. (`PR#1042 <https://github.com/terrapower/armi/pull/1042>`_)
+#. Added 4614 nuclides to decouple the loading of RIPL-3 data from the standard framework run. (`PR#998 <https://github.com/terrapower/armi/pull/998>`_)
+#. Overhaul of the tight coupling routine in ARMI, and removal of ``looseCoupling`` setting. (`https://github.com/terrapower/armi/pull/1033`_)
```
Comment (opotowsky) [inline doc/release/0.2.rst:31]:
```suggestion
#. Overhaul of the tight coupling routine in ARMI, and removal of ``looseCoupling`` setting. (`PR #1033 <https://github.com/terrapower/armi/pull/1033>`_)
```

File: doc/release/0.2.rst [hunk 5]
```diff
@@ -2,57 +2,65 @@
...
+
+
+ARMI v0.2.6
+===========
+Release Date: 2023-02-09
+
 What's new in ARMI
 ------------------
 #. The ``Material`` class no longer subclasses ``Composite`` (`PR#1062 <https://github.com/terrapower/armi/pull/1062>`_)
+#. Froze the NumPy version to <= 1.23.5 (`PR#1035 https://github.com/terrapower/armi/pull/1035`_) to continue to support NumPy jagged arrays in the DatabaseInterface.
```
Comment (opotowsky) [inline doc/release/0.2.rst:27]:
```suggestion
#. Froze the NumPy version to <= 1.23.5 (`PR#1035 <https://github.com/terrapower/armi/pull/1035>`_) to continue to support NumPy jagged arrays in the DatabaseInterface.
```

File: doc/release/0.2.rst (hunk 3 above)
Comment (opotowsky) [inline doc/release/0.2.rst:45]:
```suggestion
#. Calculate block ``PuFrac`` on core loading and after shuffling. (`PR#1165 <https://github.com/terrapower/armi/pull/1165>`_)
```

File: doc/release/0.2.rst [hunk 6]
```diff
@@ -2,57 +2,65 @@
...
-#. Bug fix to ensure consistency between cross section group manager and lattice physics interface for tight coupling. (`PR#1118 <https://github.com/terrapower/armi/pull/1118>`_)
-#. Bug fix to update pseudo density to physical density in densityTimesHeatCapacity materials method. (`PR#1129 <https://github.com/terrapower/armi/pull/1129>`_)
-#. Fixed a bug in which the TD_frac material modification on UraniumOxide and MOX materials was not being applied correctly in density calculations
-#. Added hotChannelFactors assembly parameter to the DB so that snapshot runs know about the parameter. (`PR#1157 <https://github.com/terrapower/armi/pull/1157>`_)
+#. Fixed ``referenceBlockAxialMesh`` and ``axialMesh`` during process loading. (`PR#980 <https://github.com/terrapower/armi/pull/980>`_)
+#. Fixed deadelines in MPI cases due to barriers in temp directory changers.
+#. Fixed the material namespace order for ``test_axialExpansionChanger.py`` persisting after tests. (`PR#1046 https://github.com/terrapower/armi/pull/1046`_)
```
Comment (opotowsky) [inline doc/release/0.2.rst:51]:
```suggestion
#. Fixed the material namespace order for ``test_axialExpansionChanger.py`` persisting after tests. (`PR#1046 <https://github.com/terrapower/armi/pull/1046>`_)
```

File: doc/release/0.2.rst (hunk 2 above)
Comment (opotowsky) [inline doc/release/0.2.rst:53]:
```suggestion
#. Fixed non-fuel depletable components not being initialized with all nuclides with the ``explicitFissionProducts`` model (`PR#1067 <https://github.com/terrapower/armi/pull/1067>`_)
```

File: doc/release/0.2.rst [hunk 7]
```diff
@@ -2,57 +2,65 @@
...
+#. Fixed consistency between cross section group manager and lattice physics interface for tight coupling. (`PR#1118 <https://github.com/terrapower/armi/pull/1118>`_)
+#. Fixed numerical diffusion in uniform mesh converter that affects number densities and cumulative parameters like DPA. (`PR#992 <https://github.com/terrapower/armi/pull/992>`_)
+#. Fix the formula to calculate ``b.p.puFrac``. (`PR#1168 https://github.com/terrapower/armi/pull/1168`_)
+#. Fixed ``Material.densityTimesHeatCapacity()``, moving from pseudo-density to physical density. (`PR#1129 <https://github.com/terrapower/armi/pull/1129>`_)
+#. Fixed ``TD_frac`` modification on UraniumOxide and MOX was not being applied correctly.
+#. Fixed Magnessium density curve. (`PR#1126 https://github.com/terrapower/armi/pull/1126`_)
+#. Fixed Potassium density curve. (`PR#1128 https://github.com/terrapower/armi/pull/1128`_)
+#. Fixed Concrete density curve. (`PR#1131 https://github.com/terrapower/armi/pull/1131`_)
+#. Fixed Copper density curve. (`PR#1150 https://github.com/terrapower/armi/pull/1150`_)
+#. Fixed Component.density. (`PR#1149 https://github.com/terrapower/armi/pull/1149`_)
```
Comment (opotowsky) [inline doc/release/0.2.rst:63]:
```suggestion
#. Fixed Magnessium density curve. (`PR#1126 <https://github.com/terrapower/armi/pull/1126>`_)
#. Fixed Potassium density curve. (`PR#1128 <https://github.com/terrapower/armi/pull/1128>`_)
#. Fixed Concrete density curve. (`PR#1131 <https://github.com/terrapower/armi/pull/1131>`_)
#. Fixed Copper density curve. (`PR#1150 <https://github.com/terrapower/armi/pull/1150>`_)
#. Fixed Component.density. (`PR#1149 <https://github.com/terrapower/armi/pull/1149>`_)
```

File: doc/release/0.2.rst (hunk 1 above)
Comment (opotowsky) [reply doc/release/0.2.rst:32]: good catch!

Comment (opotowsky) [reply doc/release/0.2.rst:32]:
```suggestion
#. Added ``savePhysicsFiles`` setting to copy physics kernel I/O to directories organized by cycle and time step (e.g., c2n1). (`PR#952 <https://github.com/terrapower/armi/pull/952>`_)
```

File: doc/release/0.2.rst [hunk 8]
```diff
@@ -2,57 +2,65 @@
...
+#. Fixed deadelines in MPI cases due to barriers in temp directory changers.
+#. Fixed the material namespace order for ``test_axialExpansionChanger.py`` persisting after tests. (`PR#1046 <https://github.com/terrapower/armi/pull/1046>`_)
+#. Fixed the gaseous fission products not being removed from the core directly, but instead the fission yields within the lumped fission products were being adjusted. (`PR#1022 <https://github.com/terrapower/armi/pull/1022>`_)
+#. Fixed non-fuel depletable components not being initialized with all nuclides with the ``explicitFissionProducts`` model (`PR#1067 <https://github.com/terrapower/armi/pull/1067>`_)
+#. Fixed consistency between cross section group manager and lattice physics interface for tight coupling. (`PR#1118 <https://github.com/terrapower/armi/pull/1118>`_)
+#. Fixed numerical diffusion in uniform mesh converter that affects number densities and cumulative parameters like DPA. (`PR#992 <https://github.com/terrapower/armi/pull/992>`_)
+#. Fix the formula to calculate ``b.p.puFrac``. (`PR#1168 https://github.com/terrapower/armi/pull/1168`_)
```
Comment (opotowsky) [inline doc/release/0.2.rst:56]:
```suggestion
#. Fix the formula to calculate ``b.p.puFrac``. (`PR#1168 <https://github.com/terrapower/armi/pull/1168>`_)
```

**Review by opotowsky — APPROVED** [review]
I think I missed one formatting mistake in my last sweep, but otherwise looks good to me!

**Comment by john-science** [comment]
@albeanth Thoughts?

**Review by albeanth — APPROVED** [review]

**Pull request merged**